Makefile

```diff
@@ -175,3 +186,28 @@ lint-python:
 	pip install -q -r python/test-requirements.txt
 	LC_ALL=C.UTF-8 LANG=C.UTF-8 black --config python/pyproject.toml python/
 	cd python && pylint -j4 `find . -maxdepth 1 -mindepth 1 -type d -o -name "*.py"`
+
+.PHONY: devstack
+devstack:
```

Comment:
Too much code for a Makefile.  Put this in a script and have the Makefile simply call it.

Reply:
> Too much code for a Makefile. Put this in a script and have the Makefile simply call it.

Moved to scripts/make-devstack-image and scripts/run-devstack-image.